core/wiki/macros/CSS.tid

```diff
@@ -2,7 +2,13 @@ title: $:/core/macros/CSS
 tags: $:/tags/Macro
 
 \define colour(name)
-<$transclude tiddler={{$:/palette}} index="$name$"><$transclude tiddler="$:/palettes/Vanilla" index="$name$"><$transclude tiddler="$:/config/DefaultColourMappings/$name$"/></$transclude></$transclude>
```

Comment:
I don't think it would be backwards compatible to introduce all that whitespace into the macro. Better to use `\whitespace trim`